tests/infrastructure/downward-metrics.go

```diff
@@ -23,33 +23,39 @@ import (
 	"context"
 	"time"
 
-	"kubevirt.io/kubevirt/tests/libinfra"
-	"kubevirt.io/kubevirt/tests/libvmops"
-
-	"kubevirt.io/kubevirt/tests/framework/kubevirt"
-
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+
+	k8sv1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	v1 "kubevirt.io/api/core/v1"
 	"kubevirt.io/client-go/kubecli"
 
+	"kubevirt.io/kubevirt/pkg/controller"
 	"kubevirt.io/kubevirt/pkg/libvmi"
-
-	"kubevirt.io/kubevirt/tests/libvmifact"
-
+	"kubevirt.io/kubevirt/pkg/virt-controller/watch/common"
+	"kubevirt.io/kubevirt/tests"
 	"kubevirt.io/kubevirt/tests/console"
+	"kubevirt.io/kubevirt/tests/framework/kubevirt"
+	"kubevirt.io/kubevirt/tests/framework/matcher"
+	"kubevirt.io/kubevirt/tests/libinfra"
+	"kubevirt.io/kubevirt/tests/libvmifact"
+	"kubevirt.io/kubevirt/tests/libvmops"
+	"kubevirt.io/kubevirt/tests/testsuite"
 )
 
 var _ = DescribeInfra("downwardMetrics", func() {
 	var (
 		virtClient kubecli.KubevirtClient
 	)
+
 	BeforeEach(func() {
 		virtClient = kubevirt.Client()
+		tests.EnableDownwardMetrics(virtClient)
```

Comment:
Cleanup and reset it back to its original value after? Does this need to be serial to not interfere with other tests?

Reply:
I see. Still it looks strange to me that we enable DownwardMetrics globally in the test setup and then again in the BeforeEach of these tests.

Reply:
Can you please ensure to only add the calls when needed?